packages/appsync-modelgen-plugin/src/visitors/appsync-dart-visitor.ts

```diff
@@ -361,20 +368,28 @@ export class AppSyncModelDartVisitor<
     }
     //other getters
     if (this.isNullSafety()) {
+      let forceCastException = `throw new DataStoreException(
+      DataStoreExceptionMessages.codeGenRequiredFieldForceCastExceptionMessage,
+      recoverySuggestion:
+        DataStoreExceptionMessages.codeGenRequiredFieldForceCastRecoverySuggestion,
+      underlyingException: e.toString()
+      );`;
+      if (this.config.dartUpdateAmplifyCoreDependency === true) {
```

Comment:
Just want to confirm if this exception change is expected along with flutter zero four. Otherwise LGTM 🥇

Reply:
Yes these exceptions are different in `amplify_core` package. @ragingsquirrel3 can confirm.